azkaban-web-server/src/main/java/azkaban/webapp/servlet/ProjectManagerServlet.java

```diff
@@ -1761,6 +1761,20 @@ private void ajaxHandleUpload(final HttpServletRequest req, final HttpServletRes
       throws ServletException, IOException {
     final User user = session.getUser();
     final String projectName = (String) multipart.get("project");
+
```

Comment:
I sense a bit of a code duplication... Any chance to evict it to a shared/common util method?

Reply:
I added this in Reportal much later. I agree. Let me explore it and see if it can be done.